src/rendering/graphics/shared/Graphics.ts

```diff
@@ -34,4 +38,242 @@ export class Graphics extends Container<GraphicsView>
+    }
+    public svg(...args: Parameters<GraphicsContext['svg']>): this
+    {
+        return this._callContextMethod('svg', args);
+    }
+    public restore(...args: Parameters<GraphicsContext['restore']>): this
+    {
+        return this._callContextMethod('restore', args);
+    }
+    public save(...args: Parameters<GraphicsContext['save']>): this
+    {
+        return this._callContextMethod('save', args);
+    }
+    public getTransform(...args: Parameters<GraphicsContext['getTransform']>): this
+    {
+        return this._callContextMethod('getTransform', args);
+    }
+    public resetTransform(...args: Parameters<GraphicsContext['resetTransform']>): this
+    {
+        return this._callContextMethod('resetTransform', args);
+    }
+    public rotateTransform(...args: Parameters<GraphicsContext['rotate']>): this
+    {
+        return this._callContextMethod('rotate', args);
+    }
+    public scaleTransform(...args: Parameters<GraphicsContext['scale']>): this
+    {
+        return this._callContextMethod('scale', args);
+    }
+    public setTransform(transform: Matrix): this;
+    public setTransform(a: number, b: number, c: number, d: number, dx: number, dy: number): this;
+    public setTransform(a: number | Matrix, b?: number, c?: number, d?: number, dx?: number, dy?: number): this;
+    public setTransform(...args: [Matrix] | [number, number, number, number, number, number]): this
+    {
+        return this._callContextMethod('setTransform', args);
+    }
+    public transform(transform: Matrix): this;
+    public transform(a: number, b: number, c: number, d: number, dx: number, dy: number): this;
+    public transform(a: number | Matrix, b?: number, c?: number, d?: number, dx?: number, dy?: number): this;
+    public transform(...args: [Matrix] | [number, number, number, number, number, number]): this
+    {
+        return this._callContextMethod('transform', args);
+    }
+    public translateTransform(...args: Parameters<GraphicsContext['translate']>): this
+    {
+        return this._callContextMethod('translate', args);
+    }
+    public clear(...args: Parameters<GraphicsContext['clear']>): this
+    {
+        return this._callContextMethod('clear', args);
+    }
+    get fillStyle(): GraphicsContext['fillStyle']
+    {
+        return this.view.context.fillStyle;
+    }
+    set fillStyle(value: GraphicsContext['fillStyle'])
+    {
+        this.view.context.fillStyle = value;
+    }
+    get strokeStyle(): GraphicsContext['strokeStyle']
+    {
+        return this.view.context.strokeStyle;
+    }
+    set strokeStyle(value: GraphicsContext['strokeStyle'])
+    {
+        this.view.context.strokeStyle = value;
+    }
+
+    // v7 deprecations
+    /**
+     * @param {...any} args
+     * @deprecated since 8.0.0
+     */
+    public drawCircle(...args: Parameters<GraphicsContext['circle']>): this
+    {
+        deprecation('8.0.0', 'Graphics#drawCircle has been renamed to Graphics#circle');
```

Comment:
I don't know how you set it up in v8, but in v7, I'm removing deprecation logs (#9637 9528) from the release build by doing this:

```js
if (process.env.DEBUG)
{
  deprecation('8.0.0', '...');
}
```

Reply:
Hey @bigtimebuddy , you think if we added process.env.DEBUG into the deprecation function.. it would see it as dead code and just remove the function?

Reply:
Sure, that would remove the actual console log, but we actually want to remove the deprecation call itself or else we are inlining a bunch of strings that don't do anything.